Tests/LogicTests/SantaCacheTest.mm

```diff
@@ -81,6 +83,46 @@ - (void)testCacheResetAtLimit {
   XCTAssertEqual(sut.get(6), 42);
 }
 
+- (void)testDistribution {
+  const int bucket_ratio = 5;
+  auto sut = SantaCache<uint64_t, uint64_t>(5000, bucket_ratio);
+
+  // Fill the cache with random keys, all set to 1.
```

Comment:
It might be a good idea to have the test keys as a parameter.

Reply:
Done, sort of.